### ISSUE TITLE
how to add addons

### ISSUE THREAD

**Issue by mathpopo** [openframeworks/openFrameworks#8031]
ubuntu22.04
cd OF/examples/graphics/polygonExample
make
make run

but no make

**Comment by dimitre** [comment]
try
```
cd OF/examples/graphics/polygonExample
cp ../scripts/templates/linux/{Makefile,config.make} .
make -j
make RunRelease
```

**Comment by mathpopo** [comment]
![image](https://github.com/openframeworks/openFrameworks/assets/21274466/d9f956ad-16af-4084-95e2-aa75c3654b06)
ok @dimitre ，thanks! any other example need do like this?

**Comment by mathpopo** [comment]
@dimitre if i want to add new addons, should  i do ?

**Comment by dimitre** [comment]
you can edit ```addons.make``` file in project folder and add there
take a look at ```examples/templates/allAddonsExample```

**Comment by mathpopo** [comment]
for example :
first:
~/disk1/github/openFrameworks/addons/ofxTrueTypeFontUL2$ ls
example  libs  LICENSE.md  README.md  src
senond:

openFrameworks/examples/templates/allAddonsExample$ ls
addons.make  allAddonsExample.png  bin  README.md  src
------------------------------------------------------------------------------------
add  ofxTrueTypeFontUL2
![image](https://github.com/openframeworks/openFrameworks/assets/21274466/a8232784-270b-431a-a527-1468f7544d10)

above all done!!!

again download_libs.sh && compileOF.sh && compilePG.sh?

**Comment by dimitre** [comment]
no need to run this scripts. I think this two are enough
```
make -j
make RunRelease
```

**Comment by mathpopo** [comment]
i just want to use file(GLOB OF_LIBS
"${OF_ROOT}/libs/openFrameworksCompiled/lib/linux64/.a"
"${OF_ROOT}/addons/**/libs//lib/linux64/.a"
"${OF_ROOT}/libs/tess2/lib/linux64/.a"
)
in which folder to run "make"?~/disk1/github/openFrameworks/examples/templates/allAddonsExample$ ?

**Comment by dimitre** [comment]
go to the example/project folder and run
```
cp ../scripts/templates/linux/{Makefile,config.make} .
```
so you have the default makefiles in this folder and can run make

**Comment by dimitre** [comment]
Yes this addon probably have to be modified to run corrrectly with latest openFrameworks

**Comment by mathpopo** [comment]
could you give me a ofx like ofxpango?

**Comment by 2bbb** [comment]
@mathpop 
I think it's best to ask on the forum.
https://forum.openframeworks.cc

**Comment by danoli3** [comment]
I reckon we can do better with addons / management. I know there's a few solutions but if we can cook a solution into the c++ command line project generator and thus embed it also in the gui I think we will solve a lot of hassles. I'll side quest into project generator stuff while I sort out Android merge. 

Lots of blocks we need to fix for the devs

**Comment by dimitre** [comment]
Hello @mathpopo I'll be closing this one because the main issue was answered
I think you are using an outdated addon so some fix is needed, you can open an issue on author repository
Hope you insist in OF and have your issues fixed!
